Index: libltdl/acinclude.m4
===================================================================
RCS file: /cvs/kaffe/kaffe/libltdl/acinclude.m4,v
retrieving revision 1.33
diff -u -r1.33 acinclude.m4
--- libltdl/acinclude.m4	20 Sep 2004 23:19:07 -0000	1.33
+++ libltdl/acinclude.m4	7 Oct 2004 22:47:46 -0000
@@ -1522,7 +1522,11 @@
 openbsd*)
   version_type=sunos
   need_lib_prefix=no
-  need_version=no
+  # Some older versions of OpenBSD (3.3 at least) *do* need versioned libs.
+  case $host_os in
+    openbsd3.3 | openbsd3.3.*)	need_version=yes ;;
+    *)				need_version=no  ;;
+  esac
   library_names_spec='${libname}${release}${shared_ext}$versuffix ${libname}${shared_ext}$versuffix'
   finish_cmds='PATH="\$PATH:/sbin" ldconfig -m $libdir'
   shlibpath_var=LD_LIBRARY_PATH
