Index: libltdl/Makefile.am
===================================================================
RCS file: /cvs/kaffe/kaffe/libltdl/Makefile.am,v
retrieving revision 1.19
diff -u -r1.19 Makefile.am
--- libltdl/Makefile.am	27 Apr 2005 08:50:45 -0000	1.19
+++ libltdl/Makefile.am	28 Apr 2005 09:26:55 -0000
@@ -31,6 +31,7 @@
 libtool: $(LIBTOOL_DEPS)
 	$(SHELL) ./config.status --recheck
 
+if INSTALL_LTDL
 ## These are installed as a subdirectory of pkgdatadir so that
 ## libtoolize --ltdl can find them later:
 ltdldatadir = $(datadir)/libtool/libltdl
@@ -47,3 +48,5 @@
 	$(mkinstalldirs) $(DESTDIR)$(ltdldatadir)
 	( cd $(srcdir) && $(AMTAR) chf - $(ltdldatafiles); ) \
 	  | ( umask 0 && cd $(DESTDIR)$(ltdldatadir) && $(AMTAR) xf -; )
+endif
+
