Index: libltdl/acinclude.m4
===================================================================
RCS file: /cvs/kaffe/kaffe/libltdl/acinclude.m4,v
retrieving revision 1.30
diff -u -r1.30 acinclude.m4
--- libltdl/acinclude.m4	17 Apr 2004 10:31:14 -0000	1.30
+++ libltdl/acinclude.m4	21 Jul 2004 22:43:47 -0000
@@ -317,8 +317,8 @@
 
 # The HP-UX ksh and POSIX shell print the target directory to stdout
 # if CDPATH is set.
-if test "X${CDPATH+set}" = Xset; then CDPATH=:; export CDPATH; fi
-
+#if test "X${CDPATH+set}" = Xset; then CDPATH=:; export CDPATH; fi
+(unset CDPATH) >/dev/null 2>&1 && unset CDPATH
 if test -z "$ECHO"; then
 if test "X${echo_test_string+set}" != Xset; then
 # find a string as large as possible, as long as the shell can cope with it
@@ -3971,7 +3971,7 @@
 # The HP-UX ksh and POSIX shell print the target directory to stdout
 # if CDPATH is set.
 if test "X\${CDPATH+set}" = Xset; then CDPATH=:; export CDPATH; fi
-
+(unset CDPATH) >/dev/null 2>&1 && unset CDPATH 
 # The names of the tagged configurations supported by this script.
 available_tags=
 
Index: libltdl/ltmain.sh
===================================================================
RCS file: /cvs/kaffe/kaffe/libltdl/ltmain.sh,v
retrieving revision 1.4
diff -u -r1.4 ltmain.sh
--- libltdl/ltmain.sh	15 Apr 2004 20:58:16 -0000	1.4
+++ libltdl/ltmain.sh	21 Jul 2004 22:43:48 -0000
@@ -4811,8 +4811,8 @@
 
 # The HP-UX ksh and POSIX shell print the target directory to stdout
 # if CDPATH is set.
-if test \"\${CDPATH+set}\" = set; then CDPATH=:; export CDPATH; fi
-
+#if test \"\${CDPATH+set}\" = set; then CDPATH=:; export CDPATH; fi
+(unset CDPATH) >/dev/null 2>&1 && unset CDPATH 
 relink_command=\"$relink_command\"
 
 # This environment variable determines our operation mode.
